### DISCUSSION TITLE
[Feature Request] ScrollShadow component should display shadow in a proportional size

### DISCUSSION THREAD

**Discussion by keul** [heroui-inc/heroui#4722] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently the ScrollShadow component behaves as a "all or none": if use reach the top/bottom (or left/right) of the scroll, the shadown of the intended size is displayed.

This is not very nice to see, as a 1px scroll makes the whole shadow to appear in

### Describe the solution you'd like

https://lea.verou.me/blog/2012/04/background-attachment-local/

In the example above, the scroll shadow fade-in/out slowly, based on the scroll position.

### Describe alternatives you've considered

https://lea.verou.me/blog/2012/04/background-attachment-local/

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1750/[feature-request]-scrollshadow-component-should-display-shadow-in-a">ENG-1750 [Feature Request] ScrollShadow component should display shadow in a proportional size</a></p>
